crt_portal/static/sass/custom/table.scss

```diff
@@ -14,4 +14,33 @@
       background-color: $white;
     }
   }
-}
\ No newline at end of file
+}
+
+// Styles for complaint statuses in View All Table
+
+.status-new {
+  color: $white !important;
+  background-color: $blue-warm-vivid-70;
+  padding: 4px;
+  padding-left: 10px;
+  padding-right: 10px;
+  border-radius: 1.5rem;
+}
+
+.status-open {
+  background-color: $gold-20v;
```

Comment:
We need to add this to the colors as well

     $gold-20v: #FFBE2E;

Reply:
I noticed that in `colors.scss` we had defined:

```
$gold: #FFBE2E;
```

Since these are the same values I think we can just use a single `$gold` for now.

Reply:
This compiles correctly for me. If you feel strongly about the variable name, we can get more specific than `$gold` — but elsewhere we're spelling out "vivid" as in `$blue-vivid-40`, so I'd want to keep that consistent.